tests/parametric/test_telemetry.py

```diff
@@ -89,6 +89,93 @@ def test_library_settings(self, library_env, test_agent, test_library):
             assert cfg_item.get("origin") == "default", "Unexpected origin for '{}'".format(apm_telemetry_name)
 
 
+@scenarios.parametric
+@rfc("https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o")
+@features.telemetry_app_started_event
+# To pass this test, ensure the lang you are testing has the necessary mapping in its config_rules.json file: https://github.com/DataDog/dd-go/tree/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static
+# And replace the `missing_feature` marker under the lang's manifest file, for Test_Consistent_Configs
+class Test_Consistent_Configs:
+    """Clients should report modifications to features."""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                # Decrease the heartbeat/poll intervals to speed up the tests
+                "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.1",
+                # Multiple integrations disabled to capture compatibility across tracers
+                "DD_TRACE_GRPC_ENABLED": "false",  # applies to python, java, dotnet, ruby, node
+                "DD_TRACE_PHPREDIS_ENABLED": "false",  # applies to php only
+                "DD_TRACE_RATE_LIMIT": 100,
+                "DD_TRACE_HEADER_TAGS": "header:tag",
+                "DD_TRACE_ENABLED": "true",
+                "DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP": "^[a-zA-Z]$",
+                "DD_TRACE_LOG_DIRECTORY": "/some/temporary/directory",
+                "DD_VERSION": "123",
+                "DD_HTTP_CLIENT_ERROR_STATUSES": "400",
+                "DD_HTTP_SERVER_ERROR_STATUSES": "500",
+                "DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "true",
+                "DD_TRACE_CLIENT_IP_HEADER": "X-Forwarded-For",
+                "DD_TRACE_SERVICE_MAPPING": "plugin:custom",
```

Comment:
Can we remove DD_TRACE_SERVICE_MAPPING for now since that's now out-of-scope for this push?